pkgs/data/fonts/kacst/default.nix

```
@@ -0,0 +1,22 @@
+{ stdenv, fetchurl, lib }:
+
+stdenv.mkDerivation {
```

This should fix the `ofborg-eval` error:
- https://gist.github.com/d9e9aa333422ecd0d134b69d96859663

```suggestion
stdenv.mkDerivation rec {
```